cmd/storj-sim/network.go

```diff
@@ -232,7 +233,7 @@ func newNetwork(flags *Flags) (*Processes, error) {
 	var satellites []*Process
 	for i := 0; i < flags.SatelliteCount; i++ {
 		process := processes.New(Info{
-			Name:       fmt.Sprintf("satellite/%d", i),
+			Name:       fmt.Sprintf("satellite-api/%d", i),
```

Comment:
We need to keep this as `satellite/%d`, because otherwise it could break third-party scripts and uses of `storj-sim`.

Reply:
oh ok i didn't know that, will fix